packages/expo-modules-core/ios/Tests/ExpoModulesSpec.swift

```diff
@@ -0,0 +1,72 @@
+// Copyright 2022-present 650 Industries. All rights reserved.
+
+import ExpoModulesTestCore
+
+@testable import ExpoModulesCore
+
+class ExpoModulesSpec: ExpoSpec {
+  override func spec() {
+    let appContext = AppContext.create()
+    let interopBridge = SwiftInteropBridge(appContext: appContext)
+    let runtime = appContext.runtime
+    let testModuleName = "TestModule"
+    let testFunctionName = "testFunction"
+    let constantsDict: [String: Any] = [
+      "expo": "is cool",
+      "sdk": 45,
```

Comment:
Maybe we should keep track of this value, it needs to be updated with every SDK cycle 😆